Client/Ecosia/UI/MyImpact/MultiplyImpactStep.swift

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+final class MultiplyImpactStep: UIView, Themeable {
+    private weak var indicator: UIView?
+    private weak var titleLabel: UILabel?
+    private weak var subtitleLabel: UILabel?
+    
+    required init?(coder: NSCoder) { nil }
+    init(title: String, subtitle: String) {
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = false
+        isUserInteractionEnabled = false
+        
+        let indicator = UIView()
+        indicator.translatesAutoresizingMaskIntoConstraints = false
+        indicator.isUserInteractionEnabled = false
+        indicator.clipsToBounds = true
+        indicator.layer.cornerRadius = 6
+        addSubview(indicator)
+        self.indicator = indicator
+        
+        let titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.numberOfLines = 0
+        titleLabel.text = title
+        titleLabel.font = .preferredFont(forTextStyle: .subheadline)
```

Comment:
FYI: to have a fully dynamic effect on runtime, you would need to add a line like:
`titleLabel.adjustsFontForContentSizeCategory = true`

otherwise it will only adjust after restarting the app

Reply:
Thanks for the reminder, just added them.